keybase/platform_windows.go

```diff
@@ -34,6 +34,7 @@ type guid struct {
 // F1B32785-6FBA-4FCF-9D55-7B8E7F157091
```

Comment:
Please move this comment to before the `folderIDLocalAppData` constant, and make an equivalent for `folderIDRoamingAppData`.